indico/modules/logs/controllers.py

```diff
@@ -159,3 +161,34 @@ def object(self):
     @property
     def object_tzinfo(self):
         return get_timezone(config.DEFAULT_TIMEZONE)
+
+
+class RHResendEmail(RHManageEventBase):
+    """Resend an email log entry."""
+
+    def _process_args(self):
+        self.entry = EventLogEntry.get(request.view_args['log_entry_id'])
+        if self.entry is None:
+            raise NotFound('Event log entry does not exist.')
+        return super()._process_args()
+
+    def _process(self):
+        if self.entry.type != 'email':
+            raise Forbidden('Log entry is not an email.')
+        email = make_email(
+            to_list=self.entry.data['to'],
+            cc_list=self.entry.data.get('cc', []),
+            bcc_list=self.entry.data.get('bcc', []),
+            sender_address=self.entry.data['from'],
+            reply_address=self.entry.data.get('reply_to', []),
+            subject=self.entry.data['subject'],
+            body=self.entry.data['body'],
+            html=self.entry.data.get('content_type') == 'text/html',
```

Comment:
I think you can use bracket notation everywhere since these keys should always exist